ui/components/schema/SchemaContainer.js

```diff
@@ -19,16 +19,25 @@ class SchemaContainer extends Component {
     constructor(props) {
         super(props);
         this.state = INITIAL_STATE;
+        this.flexWrapper = null;
+
+        this.setflexWrapperRef = e => {
```

Comment:
The new way to create refs in react is to use `React.createRef()`: https://reactjs.org/docs/refs-and-the-dom.html#creating-refs